bentoml/_internal/configuration/containers.py

```diff
@@ -434,5 +442,32 @@ def tracing_excluded_urls(
     remote_runner_mapping = providers.Static[t.Dict[str, str]]({})
     plasma_db = providers.Static[t.Optional["ext.PlasmaClient"]](None)
 
+    @providers.SingletonFactory
+    @staticmethod
+    def duration_buckets(
+        metrics: t.Dict[str, t.Any] = Provide[config.api_server.metrics],
+    ) -> t.Tuple[float, ...]:
```

Comment:
```suggestion
        metrics: dict[str, t.Any] = Provide[config.api_server.metrics],
    ) -> tuple[float, ...]:
```

Reply:
What's our standards on using `__future__` annotations?

Reply:
We should just use it from now on right?